lib/ohai/plugins/linux/platform.rb

```diff
@@ -139,7 +139,7 @@ def determine_platform_family
       "amazon"
     when /suse/, /sles/, /opensuse/
       "suse"
-    when /fedora/, /pidora/, /arista_eos/
+    when /fedora/, /pidora/, /arista_eos/, /xcp/
```

Comment:
Looking at the xcp docs it looks like we should actually classify this as RHEL. Can you change that back and then we can merge this.

Also can you open a PR against https://github.com/chef/os_release. We're updating ohai to use the /etc/os-release file for system detection and it will help us to have an idea what this system looks like.